pkg/sdn/api/v1/types.go

```diff
@@ -80,3 +80,50 @@ type NetNamespaceList struct {
 	// Items is the list of net namespaces
 	Items []NetNamespace `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+// EgressNetworkPolicyRuleType gives the type of an EgressNetworkPolicyRule
+type EgressNetworkPolicyRuleType string
+
+const (
+	EgressNetworkPolicyRuleAllow EgressNetworkPolicyRuleType = "Allow"
+	EgressNetworkPolicyRuleDeny  EgressNetworkPolicyRuleType = "Deny"
+)
+
+// EgressNetworkPolicyPeer specifies a target to apply egress policy to
+type EgressNetworkPolicyPeer struct {
+	// CIDRSelector is the CIDR range to allow/deny traffic to
```

Comment:
Use the external values in the godoc on public API fields so that swagger has the correct name (someone seeing swagger doesn't use go, they use JSON), so for all of these `cidrSelector is the CIDR range`